platform/lib/routers/growPages.js

```diff
@@ -291,6 +299,9 @@ growPages.get(/^(.*\/)?([^\/\.]+|.+\.html|.*\/|$)$/, async (req, res, next) => {
     signale.error('[OPTIMIZER]', e);
   }
 
+  // Cache the optimized and rendered page
+  pageCache.set(req.originalUrl, renderedTemplate);
```

Comment:
Do this after sending the page content

Reply:
Don't we need to encode the current version into the cache key?

Reply:
Yep that's done in `pageCache` itself.